docs/index.md

```diff
@@ -45,16 +45,7 @@ This project was initially inspired by a [gist](https://gist.github.com/mosheesh
 
 ## Contributing
 
-* Star the project on [Github](https://github.com/testcontainers/testcontainers-java) and help spread the word :)
-* Join our Slack: http://slack.testcontainers.org
-* See [ROADMAP](ROADMAP.md) to understand the approach behind the project and what may/may not be in store for the future.
-* [Post an issue](https://github.com/testcontainers/testcontainers-java/issues) if you find any bugs
-* Contribute improvements or fixes using a [Pull Request](https://github.com/testcontainers/testcontainers-java/pulls). If you're going to contribute, thank you! Please just be sure to:
-	* discuss with the authors on an issue ticket prior to doing anything big
-	* follow the style, naming and structure conventions of the rest of the project
-	* make commits atomic and easy to merge
-	* verify all tests are passing. Build the project with `./gradlew check` to do this.
-	**N.B.** Gradle's Build Cache is enabled by default, but you can add `--no-build-cache` flag to disable it.
+See [CONTRIBUTING](../CONTRIBUTING.md).
```

Comment:
Probably will not work on Gitbook. Could you plz double check it locally? (you can run gitbook on your machine)

Reply:
I see - that's a shame. I've updated to something that will work :)